api/src/routes/policies/instance/domains.ts

```diff
@@ -0,0 +1,16 @@
+import { Router, Request, Response } from "express";
+import { route } from "@fosscord/api";
+import { Config } from "@fosscord/util";
+import { config } from "dotenv"
+const router = Router();
+const { cdn } = Config.get();
+
+const IdentityForm = {
+    cdn: cdn.endpointPublic || process.env.CDN || "http://localhost:3001",
```

Comment:
add gateway

Reply:
`/gateway` route already handles that but I will add